### DISCUSSION TITLE
İbax Tüm zamanların en iyi ölçeklenebiliri olabilir

### DISCUSSION THREAD

**Discussion by Erkal45** [IBAX-io/go-ibax#1905] [Ideas]
Avax'tan daha hızlı olabilirsin.. Fantom FTM büyük bir blok zinciri buna örnek olabilir faydalınabilir..

**Comment by caner550** [comment]
Avax balon ve boş bir sistem .
Şişirilmiş patlar .
Borsa fiyatı yüzünden tutuluyor ve hızı solana yanından gecmez .

Ibax da temeli sağlam atıyor bakalım göreceğiz
